src/main/java/de/florianmichael/viafabricplus/injection/mixin/compat/ipnext/MixinAutoRefillHandler_ItemSlotMonitor.java

```diff
@@ -32,21 +32,22 @@
  *
  * Fixes https://github.com/ViaVersion/ViaFabricPlus/issues/209
  */
+@SuppressWarnings("all")
 @Pseudo
 @Mixin(targets = "org.anti_ad.mc.ipnext.event.AutoRefillHandler$ItemSlotMonitor")
 public class MixinAutoRefillHandler_ItemSlotMonitor {
 
     @Shadow
     public int currentSlotId;
 
-    @Inject(method = { "checkHandle", "checkShouldHandle" }, at = @At("HEAD"), cancellable = true)
+    @Inject(method = { "checkHandle", "checkShouldHandle" }, at = @At("HEAD"), cancellable = true, remap = false)
```

Comment:
Just add the remap = false to the class header (Line 37) and remove it from all methods